tests/nginx-tests/tengine-tests/resolver_file.t

```diff
@@ -164,17 +156,18 @@ sub bind_daemon {
 }
 
 sub start_bind {
+    # cannot refer $t in start_bind directly, otherwise NGINX::TEST will fail
+    my ($nt) = @_;
     if (defined $bind_server_port) {
 
-        print "+ DNS server: try to bind server port: $bind_server_port\n";
+        print("+ DNS server: try to bind server port: $bind_server_port\n");
 
-        $t->run_daemon(\&bind_daemon);
```

Comment:
The reference of `$t`  will make case failed because of not running alert check and sanitizer error check.